Training Modules/01 Introduction/1.01 Say Hello World With Python/solution.py

```diff
@@ -1 +1,4 @@
-print("Hello, World!")
+# simple hello world program
```

Comment:
This change is not at all a required change and is also technically wrong since you haven't contributed to this problem.